com.weibellab.comms/Editor/reliablecommunicationeditorui.cs

```diff
@@ -18,6 +18,10 @@ public class ReliableCommunicationEditorUI : Editor
         private static string[] eventHandlerToolbar = new string[] { "Configuration", "Data Events", "Status Events" };
```

Comment:
Updated the editor

![image](https://user-images.githubusercontent.com/33668799/116825945-6bd34380-ab46-11eb-80d9-49b4222e10a7.png)
![image](https://user-images.githubusercontent.com/33668799/116825938-64ac3580-ab46-11eb-9f20-18016f474238.png)
Added a placeholder for using UDP to find other Comms instances.
![image](https://user-images.githubusercontent.com/33668799/116825952-72fa5180-ab46-11eb-85d4-5ac7b7b18485.png)